### PR TITLE
Export currency display component #55

### PR DESCRIPTION
**Pull request by tigredonorte**

### CODE REVIEW

File: libs/gdk/src/lib/gdk-import/gdk-import.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <span></span>
+  <span>// template.html</span>
+  <ng-container *ngIf="!!keys&&keys.length>0;else withoutParameters">
+    <ng-container *ngTemplateOutlet="withParameters"></ng-container>
+  </ng-container>
+  <span></span>
+</pre>
+
+<ng-template #withoutParameters>
+  <span style='margin: 0'>
```
Comment (oliveira-gust) [inline libs/gdk/src/lib/gdk-import/gdk-import.component.html]: Why use inline style?

File: libs/ui-toolkit/currency-display/src/lib/currency-display.component.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import { BigCurrencyPipe, SymbolDisplay } from '@gorilainvest/ui-toolkit/pipes';
+
+/**
+ * @class CurrencyDisplayComponent
+ * This component is used to format a number into currency format
+ *
+ * @example <gro-currency-display value="5.00"></gro-currency-display> will display: US$ 5
+ */
+@Component({
+  selector: 'gro-currency-display',
```
Comment (oliveira-gust) [inline libs/ui-toolkit/currency-display/src/lib/currency-display.component.ts]: If I'm not mistaken the prefix should be gor

File: libs/gdk/src/lib/gdk-import/gdk-import.component.html (hunk 1 above)
Comment (tigredonorte) [reply libs/gdk/src/lib/gdk-import/gdk-import.component.html]: There's no reason. It can be changed..

File: libs/gdk/src/lib/gdk-import/gdk-import.component.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<pre>
```
Comment (guilhermejcgois) [inline libs/gdk/src/lib/gdk-import/gdk-import.component.html]: why `gdk-import` wasn't done in another PR?

Comment (tigredonorte) [reply libs/gdk/src/lib/gdk-import/gdk-import.component.html]: I've create this component when I needed to manipulate html code to display the "fun area" widget. So, instead to waste time in my own component, I created another component that can be used for other person. If I tried to create in separated pull request, I needed to refactor an existent component (of shell folder). Because those components doesn't use ReactiveForms (that is requested by gdk-import).
